### PR TITLE
Update SUI nav for profiles

### PR DESCRIPTION
**Pull request by cinnamon-msft**
## Summary of the Pull Request
Create a header for Profiles inside the nav, where dynamic profiles can populate.

image

 
## PR Checklist
* [ ] Closes #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Documentation updated. If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/terminal) and link it here: #xxx
* [ ] Schema updated.
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 1]
```diff
@@ -355,7 +355,7 @@
     <value>Open JSON file</value>
   </data>
   <data name="Nav_ProfileGlobal.Content" xml:space="preserve">
-    <value>Profiles</value>
+    <value>Globals</value>
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:358]:
```suggestion
    <value>Global</value>
```
If you change it to "Globals", it still changes from the value this morning.

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 2]
```diff
@@ -652,4 +652,10 @@
     <value>⚠ You have unsaved changes.</value>
     <comment>{Locked="⚠"}</comment>
   </data>
+  <data name="Nav_ProfileAddNew.Content" xml:space="preserve">
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:655]:
```suggestion
  <data name="Nav_AddNewProfile.Content" xml:space="preserve">
```
Same reasoning as before. Just trying to align this with #8089 to reduce the burden on the localization folks. (make sure you change the uuid in the xaml file)

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 3]
```diff
@@ -652,4 +652,10 @@
     <value>⚠ You have unsaved changes.</value>
     <comment>{Locked="⚠"}</comment>
   </data>
+  <data name="Nav_ProfileAddNew.Content" xml:space="preserve">
+    <value>Add new</value>
+  </data>
+  <data name="Nav_ProfileHeader.Content" xml:space="preserve">
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:658]:
```suggestion
  <data name="Nav_Profiles.Content" xml:space="preserve">
```
Same as the other two comments.

**Review by carlos-zamora — CHANGES_REQUESTED** [review]

**Review by carlos-zamora — APPROVED** [review]
Don't forget to fix the merge conflicts

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 4]
```diff
@@ -1,683 +1,688 @@
...
+  </data>
+  <data name="Globals_LaunchSizeMaximizedFocus.Content" xml:space="preserve">
+    <value>Maximized Focus</value>
+  </data>
+  <data name="Profile_BellStyle.Header" xml:space="preserve">
+    <value>Bell notification style</value>
+  </data>
+  <data name="Profile_BellStyle.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Controls what happens when the application emits a BEL character.</value>
+
```
Comment (cinnamon-msft) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:670]:
```suggestion
```

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 5]
```diff
@@ -1,683 +1,687 @@
...
+  </data>
+  <data name="Settings_UnsavedSettingsWarning.Text" xml:space="preserve">
+    <value>⚠ You have unsaved changes.</value>
+    <comment>{Locked="⚠"}</comment>
+  </data>
+  <data name="Nav_AddNewProfile.Content" xml:space="preserve">
+    <value>Add new</value>
+  </data>
+  <data name="Nav_Profiles.Content" xml:space="preserve">
+    <value>Profiles</value>
```
Comment (cinnamon-msft) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:658]:
```suggestion
    <value>Profiles</value>
</data>
```

**Comment by DHowett** [comment]
I thought we were doing a single dropdown on a single profiles page...

**Comment by DHowett** [comment]
Making it a “button” that is a nav item seems like confusing UI. It’s not a page, like all the other things. It’s an _action_.

**Comment by carlos-zamora** [comment]
@cinnamon-msft and I tried having a dropdown for profiles, but it looked really weird. The pivot control acts as a header. So putting a dropdown above the header looked really off.

Instead, we saw that you can put a header on the navigation view. So we're doing that instead. That way we don't hit the "nested" bug.

**Pull request merged**